web/src/store/index.js

```diff
@@ -12,6 +13,7 @@ export default new Vuex.Store({
   modules: {
     girder,
     publicDandisets,
+    myDandisets,
```

Comment:
We can certainly register the same module store times under different module names: https://vuex.vuejs.org/guide/modules.html#module-reuse

Would it be possible to merge the logic of `publicDandisets` and `myDandisets` into a single store definition, and register it twice under different module names?

Reply:
Ahh good point, yeah I'll try that

Reply:
f1a87ed addresses this.